client/starwhale/base/bundle_copy.py

```diff
@@ -371,23 +421,26 @@ def _do_ubd_end(self, upload_id: str, url_path: str, ok: bool) -> None:
     def _do_upload_bundle_dir(
         self,
         progress: Progress,
-        add_data_uri_header: bool = False,
     ) -> None:
         workdir: Path = self._get_target_path(self.src_uri)
         url_path = self._get_remote_instance_rc_url()
 
-        upload_id = self._do_ubd_prepare(
+        res_data = self._do_ubd_bundle_prepare(
             progress=progress,
             workdir=workdir,
             url_path=url_path,
         )
+        upload_id: str = res_data.get("upload_id", "")
+        if not upload_id:
+            raise Exception("get invalid upload_id")
```

Comment:
`get invalid upload_id` -> `upload_id is empty` ?

Reply:
done